test/test_grade/test_integration.py

```diff
@@ -362,3 +364,24 @@ def test_config_overrides_integration():
     got = got.reindex(sorted(got.columns), axis=1)
     want = want.reindex(sorted(want.columns), axis=1)
     assert got.equals(want)
+
+
+@pytest.mark.slow
+@pytest.mark.docker
+def test_grade_summaries():
+    """
+    Checks that are grading summaries are written to the disck
+    """
+    grade(
```

Comment:
To make this test run faster, you could mock out the launch_containers function and have it return fake data like in `test_config_overrides`. (If you do so, remove both decorators from this test case.)